App/Components/PhotoGrid.js

```diff
@@ -0,0 +1,140 @@
+// @flow
```

Comment:
Just split out the UI from the data logic, turned out to not be necessary in the end, but left it.